bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,940 @@
+{
```

Comment:
<p>Nit, i think put</p><p> </p><h1>Goal of this Notebook </h1><p> </p><h1>🔍 Summary </h1><p> </p><p>ahead of introduction of Milvus and other tech is slightly better.</p><p> </p><p>That way there is a clear opening before diving into details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Moved it